src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -151,24 +149,35 @@ class PerformanceLanding extends React.Component<Props, State> {
     return stringifyQueryObject(parsed);
   }
 
-  renderHeaderButtons() {
-    const selectView = (viewKey: FilterViews) => {
-      return () => {
-        this.setState({
-          currentView: viewKey,
-        });
-      };
-    };
+  getCurrentView(): string {
+    const {location} = this.props;
+    const currentView = location.query.view as FilterViews;
+    if (Object.values(FilterViews).includes(currentView)) {
+      return currentView;
+    } else {
+      return FilterViews.ALL_TRANSACTIONS;
+    }
```

Comment:
```suggestion
    if (Object.values(FilterViews).includes(currentView)) {
      return currentView;
    } 
    return FilterViews.ALL_TRANSACTIONS;
```

You don't need an else as the if returns.